mcs/class/Mono.Compiler/SimpleJit.Metadata/MethodBody.cs

```diff
@@ -99,21 +100,25 @@ public class MethodBody {
 	int maxStack;
 	bool initLocals;
 	int localsToken;
+	IList<LocalVariableInfo> localInfo;
 
 	public byte[] Body {
 		get { return body; }
 	}
+#if FIXME_USE_SIMPLEJIT_METADATA
 	// II.25.4.4
 	const int FlagsFormatMask   = 0x03;
 	const int FlagsFatFormat 	= 0x03;
 	const int FlagsTinyFormat   = 0x02;
 	const int FlagsMoreSections = 0x08;
 	const int FlagsInitLocals   = 0x10;
+#endif
 
 	public IlIterator GetIterator () {
 		return new IlIterator (body);
 	}
 
+#if FIXME_USE_SIMPLEJIT_METADATA
```

Comment:
random comment: huh, that's an interesting way to disable code. I like it! 👍

Reply:
TBH we should probably just remove that stuff - we're not going to be doing metadata parsing in managed.  I should have just imported the `IlIterator` and not the rest of MethodBody and so on.